156 adrg update fda feedback (#162)
RConsortium/submissions-pilot3-adam: vignettes/adrg.qmd

---
title: "Analysis Data Reviewer’s Guide"
subtitle: "R Consortium  \nR Submission Pilot 3  \nADRG Template Version 2019-07-18"
format: 
  pdf:
    toc: true
    number-sections: true

>     include-before:

Reviewer: include-before:
      - '\newpage{}'

```         
  Are the values of ARM equivalent in meaning to values of TRT01P?

    Yes.
```

ACTARM versus TRT01A

```         
  If TRT01A is used, then are the values of ACTARM equivalent to values of TRT01A?

    Not applicable - ACTARM is not used.
```

Use of ADaM Treatment Variables in Analysis

```         
  Are both planned and actual treatment variables used in analysis?

    Yes. Planned treatment variables are used for study population and efficacy 
    analyses, whilst actual treatment variables are used for the safety analysis.
    All subjects received the treatment arm to which they were randomised and so
    the planned treatment is equivalent to the actual treatment for all subjects.
```

Use of ADaM Treatment Grouping Variables in Analysis

```         
  Are both planned and actual treatment grouping variables used in analysis?

    Not applicable - treatment grouping variables are not used.
```

## Use of Visit Windowing, Unscheduled Visits, and Record Selection

Was windowing used in one or more analysis datasets?

```         
  Yes
```

Were unscheduled visits used in any analyses?

```         
  Yes
```

## Imputation/Derivation Methods

> For ASTDT in ADAE, this date was converted to numeric SAS date from AE.AESTDTC. If the day component is missing, a value of '01' is used. If both the month and day are missing no imputation is performed. See define.xml.

Reviewer 2: Did we convert it to a numeric SAS date while using R?  Maybe that part could be dropped??

Author: Yes, we did.  As this was apart of the specs, we also had to cover it in the program regardless of whether or not any dates needed imputation or not.  The function `admiral::derive_vars_dtm()` handles this imputation for us.

+----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
|                                              |                                |          |        |                         |           |                                                                               |
+----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+
| ADTTE - AE Time To 1st Derm. Event Analysis  | BASIC DATA STRUCTURE           | x        | x      |                         |           | One or more records per subject per analysis parameter per analysis timepoint |
+----------------------------------------------+--------------------------------+----------+--------+-------------------------+-----------+-------------------------------------------------------------------------------+

: {tbl-colwidths="\[20,20,10,10,20,10,10\]"}

### ADSL - Subject-Level Analysis Dataset

> The subject level analysis dataset (ADSL) contains required variables for demographics, treatment groups, and population flags. In addition, it contains other baseline characteristics that were used in both safety and efficacy analyses. All patients in DM were included in ADSL. The following are the key population flags are used in analyses for patients: • SAFFL – Safety Population Flag (all patients having received any study treatment) • ITTFL – Intent-to-Treat Population Flag (all randomized patients)

Reviewer 2: were the SAFFL and ITTFL meant to be a bulleted list?

Author: Thanks for catching this.  Fixed.

### ADADAS - ADAS-COG Analysis Dataset

> ADADAS contains analysis data from the ADAS-Cog questionnaire, one of the primary efficacy endpoints. It contains one record per subject per parameter (ADAS-Cog questionnaire item) per VISIT. Visits are placed into analysis visits (represented by AVISIT and AVISITN) based on the date of the visit and the visit windows.

Reviewer 2: Did we want to talk about the issue with slotting here?  I know we reverted back to original, but was thinking pointing this out might be handy?

Author: I understand where you're coming from, but I think that this issue will be addressed outside of the Pilot 3 submission through a blog by Joe and Ning, per the recent R Consortium RSWG minutes from the time stamp of this comment.  Though we point to the more bigger issue as captured in the QC Findings. In regards to slotting/windowing, maybe we can add a sentence to mention `... visit windows. More information on the analysis visit windows can be found in the Statistical Analysis Plan`. What do you think @kaz462 ?

### ADLBC - Analysis Dataset Lab Blood Chemistry

ADLBC contains one record per lab analysis parameter, per time point, per subject. ADLBC contains lab chemistry parameters and these data are derived from the SDTM LB (Laboratory Tests) domain. Two sets of lab parameters exist in ADLBC. One set contains the standardised lab value from the LB domain and the second set contains change from previous visit relative to normal range values. In some of the summaries the derived end-of-treatment visit (AVISITN=99) is also presented.

### ADTTE - AE Time To 1st Derm. Event Analysis

ADTTE contains one observation per parameter per subject. ADTTE is specifically for safety analyses of the time to the first dermatologic adverse event. Dermatologic AEs are considered an adverse event of special interest. The key parameter used for the analysis of time to the first dermatological event is with PARAMCD of "TTDE".

{{< pagebreak >}}

# Data Conformance Summary

## Conformance Inputs

Were the analysis datasets evaluated for conformance with CDISC ADaM Validation Checks?

```         

>   Yes, Version of CDISC ADaM Validation Checks and software used: Pinnacle 21® 

Reviewer 2: With the re-submission - should get an updated P21 report just for timestamps alignment?

Author: Yes, this would be good to do. Might be good to do a rerun refresh of all the programs and outputs again for the re-submission, as well as do a P21 run. This action still open.

+==========+==============================================================================+=========+====================+=========================================================================================================================================================================================+
| AD1012   | Secondary custom variable is present but its primary variable is not present | ADSL    | 1 (50.00%)         | This is a Sponsor Extension to the ADaM Model. The VISNUMEN \[End of Trt Visit (Vis 12 or Early Term.)\] variable is a integer variable which is not related to any character variable. |
+----------+------------------------------------------------------------------------------+---------+--------------------+-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
: {tbl-colwidths="\[17,22,15,22,25\]"}

## QC Findings and Common Issues

> In this Pilot 3 study, our focus was to create ADaMs based on Pilot 1 where after ADaM generation we compared them against the analysis datasets used in Pilot 1 as a QC step. With these comparisons we listed the QC Findings with explanations as to why these findings exist. We also came across common issues throughout the ADaM generation process, which could be helpful for improvements utilising the CDISC Pilot data in the future. More details can be found in the appendix (Appendix 2 and Appendix 3).

Reviewer 2: ```suggestion
In this Pilot 3 study, our focus was to create a subset of ADaMs using R based on the CDSICPILOT data.  We compared the original CDISCPILOT ADaMs, created in SAS, as a QC step. With these comparisons we listed the QC Findings with explanations as to why these findings exist. We also came across common issues throughout the ADaM generation process, which could be helpful for improvements utilising the CDISC Pilot data in the future. More details can be found in the appendix (Appendix 2 and Appendix 3).
```

> [QC Findings](https://github.com/RConsortium/submissions-pilot3-adam/wiki/QC-Findings)

Reviewer 2: I think we should provide some on text on where this is housed and why it is housed on GitHub

Author: Alternatively, if there is a way we can bring in this wiki into the adrg markdown so that it includes it in the adrg, then we won't have to explain why it's housed in Github.

Author: I removed to links to github wiki in the adrg and brought in the content directly into the adrg instead.

[Common Issues](https://github.com/RConsortium/submissions-pilot3-adam/wiki/Common-Issues)

{{< pagebreak >}}

# Submission of Programs

## Description

The sponsor has provided all programs for analysis results. They are all created on a Linux platform using R version 4.2.3.

## ADaM Programs

> The following table contains the list of programs that generate the analysis datasets in the R Consortium R submission Pilot 3. It shows the program file name, the analysis dataset name and the label of the analysis dataset. The recommended steps to execute the analysis results using R are described in the Appendix.
> 

Reviewer 2: ```suggestion
The following table contains the list of programs that generate the analysis datasets in the Pilot 3. It shows the program file name, the analysis dataset name and the label of the analysis dataset. The recommended steps to execute the analysis results using R are described in the Appendix.

```

+------------------+---------------------------+--------------------------------------+
| **Program Name** | **Analysis Dataset Name** | **Analysis Dataset Label**           |
+==================+===========================+======================================+
| adsl.r           | adsl.xpt                  | Subject-Level Analysis Dataset       |
+------------------+---------------------------+--------------------------------------+
| adadas.r         | adas.xpt                  | ADAS-Cog Analysis                    |
+------------------+---------------------------+--------------------------------------+
| adlbc.r          | adlb.xpt                  | Analysis Dataset Lab Blood Chemistry |
+------------------+---------------------------+--------------------------------------+
| adae.r           | adae.xpt                  | Adverse Events Analysis Dataset      |
+------------------+---------------------------+--------------------------------------+
| adtte.r          | adtte.xpt                 | AE Time to 1st Derm. Event Analysis  |
+------------------+---------------------------+--------------------------------------+

> ## Analysis Output Programs

Reviewer 2: This rendering in a bad  way for me

![image](https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/8a27898d-20c6-4c06-ad50-69ce50a890d1)

Author: We may need to play around with the table widths as was done in line 229 of `adrg.qmd`. The 'Analysis Datasets' table.  We can copy this line down to the other tables to fix the widths.

+-------------------+-------------------------+-----------------------------------------------------------------------------------+

## Proprietary R Packages

+----------------------------------------------------------------------------+---------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
| **Proprietary R Package**                                                  | **Package version** | **Analysis Package Description**                                                                                                                                                                                                            |
+----------------------------------------------------------------------------+---------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+

> | [pilot3utils](https://github.com/RConsortium/submissions-pilot3-utilities) | 0.0.1               | The objective of this utility package is to support the [R Consortium R submission Pilot 3 Project](https://rconsortium.github.io/submissions-wg/). It contains all utility functions that were used in the generation of the deliverables: |
> |                                                                            |                     |                                                                                                                                                                                                                                             |

Reviewer 2: ```suggestion
| [pilot3utils](https://github.com/RConsortium/submissions-pilot3-utilities) | 0.0.2              | The objective of this utility package is to support the [R Consortium R submission Pilot 3 Project](https://rconsortium.github.io/submissions-wg/). It contains all utility functions that were used in the generation of the deliverables: |
|                                                                            |                     |                                                                                                                                                                                                                                             |
```

This is bumped up a version with the name update

+-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+

5.  Upon completion of installing packages using {renv}, please restart your R session.

> 6.  Install the {pilot3utils} package running the code below.

Reviewer 2: Choose Package Pane

<img width="735" alt="image" src="https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/45edee0b-e4d3-414c-92a7-f908c39cebcb">

Click Intall From and change from zip file
<img width="592" alt="image" src="https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/629c8fd7-bfc7-4c9e-959f-e888f1c7447d">

Select the zip file for pilots3utils
<img width="1045" alt="image" src="https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/84b909da-0523-4c96-948f-f0c24407e376">

<img width="458" alt="image" src="https://github.com/RConsortium/submissions-pilot3-adam/assets/10111024/f6a06b0d-f8d2-477e-9afc-f26c3e2904fd">

Click install

Reviewer 2: I didn't have time to record a video - ust did some screenshots.  But the process is pretty simple to install the zip file

Author: No worries, @bms63 .  Thanks for this.  I've added the steps in words without the image captures for now.

+-----------------------------+-----------------------------------------------+

## Source Data Used for Analysis Dataset Creation

The ADaM datasets were derived from SDTM version 1.2. For traceability, the SDTM is publicly available at the [PHUSE Github Repository](https://github.com/cdisc-org/sdtm-adam-pilot-project/tree/master/updated-pilot-submission-package/900172/m5/datasets/cdiscpilot01/tabulations/sdtm).

Which can be traced back to the original [CDISC SDTM & ADaM Pilot Project](https://github.com/cdisc-org/sdtm-adam-pilot-project).

{{< pagebreak >}}

# Protocol Description

## Protocol Number and Title

-   **Protocol Number:** CDISCPilot1
-   **Protocol Title:** Safety and Efficacy of the Xanomeline Transdermal Therapeutic System (TTS) in Patients with Mild to Moderate Alzheimer’s Disease

The reference documents can be found [here](https://github.com/cdisc-org/sdtm-adam-pilot-project/blob/master/updated-pilot-submission-package/900172/m5/53-clin-stud-rep/535-rep-effic-safety-stud/5351-stud-rep-contr/cdiscpilot01/cdiscpilot01.pdf).

## Protocol Design in Relation to ADaM Concepts

### Objectives:

> The objectives of the study were to evaluate the efficacy and safety of transdermal xanomeline, 50cm and 75cm, and placebo in subjects with mild to moderate Alzheimer’s disease.

Reviewer 2: I think the power 2 needs to be added for 50 and 75

Author: Fixed.

### Methodology:

> This was a prospective, randomized, multi-center, double-blind, placebo-controlled, parallel-group study. Subjects were randomized equally to placebo, xanomeline low dose, or xanomeline high dose. Subjects applied 2 patches daily and were followed for a total of 26 weeks.

Reviewer 2: ```suggestion
This was a prospective, randomized, multi-center, double-blind, placebo-controlled, parallel-group study. Subjects were randomized equally to placebo, xanomeline low dose, or xanomeline high dose. Subjects applied 2 patches daily and were followed for a total of 27 weeks.
```
